website/content/en/docs/building-operators/golang/operator-scope.md

```diff
@@ -264,7 +264,7 @@ options := ctrl.Options{
 
 // Add support for MultiNamespace set in WATCH_NAMESPACE (e.g ns1,ns2)
 if strings.Contains(watchNamespace, ",") {
-    setupLog.Infof("manager will be watching namespace %q", watchNamespace)
+    setupLog.Info(fmt.Sprintf("manager will be watching namespace %q", watchNamespace))
```

Comment:
This should look more like what the logr package [suggests](https://godoc.org/github.com/go-logr/logr):
```suggestion
    setupLog.Info("manager set up with multiple namespaces", "namespaces", watchNamespace)
```